packages/webview_flutter/ios/Classes/FLTWKNavigationDelegate.m

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FLTWKNavigationDelegate.h"
+
+@implementation FLTWKNavigationDelegate {
+  FlutterMethodChannel* _methodChannel;
+}
+
+- (instancetype)initWithChannel:(FlutterMethodChannel*)channel {
+  self = [super init];
+  if (self) {
+    _methodChannel = channel;
+  }
+  return self;
+}
+
+- (void)webView:(WKWebView*)webView
+    decidePolicyForNavigationAction:(WKNavigationAction*)navigationAction
+                    decisionHandler:(void (^)(WKNavigationActionPolicy))decisionHandler {
+  if (!self.hasDartNavigationDelegate) {
+    decisionHandler(WKNavigationActionPolicyAllow);
+    return;
+  }
+  NSDictionary* arguments = @{
+    @"url" : navigationAction.request.URL.absoluteString,
+    @"isMainFrame" : [NSNumber numberWithBool:navigationAction.targetFrame.isMainFrame]
```

Comment:
Just a suggestion, but the code looks fine. Feel free to not update this for this PR if you don't feel like to. 
The [NSNumber numberWithBool..] can be short written as @(foo).
```suggestion
    @"isMainFrame" : @(navigationAction.targetFrame.isMainFrame)
```

Reply:
sweet!